src/components/GetStartedPopover/generateGetStartedLinks.tsx

```diff
@@ -0,0 +1,128 @@
+import {
+  IconAndroid,
+  IconAngular,
+  IconFlutter,
+  IconJS,
+  IconNext,
+  IconReact,
+  IconSwift,
+  IconVue
+} from '@/components/Icons';
+import { GetStartedLinksType } from './GetStartedPopover';
+import { PLATFORM_DISPLAY_NAMES, Platforms } from '@/data/platforms';
+
+/**
+ * Generates get started links for all platforms with the same get started url
+ * @param getStartedPathname
+ * @returns {GetStartedLinksType[]} The get started link objects to be used in GetStartedPopover component
+ */
+export function generateGetStartedLinks(
+  getStartedPathname
+): GetStartedLinksType[] {
+  const platformOrder: Platforms = [
+    'react',
+    'javascript',
+    'flutter',
+    'swift',
+    'android',
+    'react-native',
+    'angular',
+    'nextjs',
+    'vue'
+  ];
```

Comment:
Added this because I realized the order from platforms: https://github.com/aws-amplify/docs/blob/911eac0c1badbd2011f0f4fe2491caf0bf05700b/src/data/platforms.ts#L23-L28

 was generating the links in a different order from our prod site. This array helps set up the order to match what we have on prod